umake/frameworks/android.py

```diff
@@ -136,8 +136,7 @@ def post_install(self):
         # the SDK manager
         add_env_to_user(self.name, {"PATH": {"value": [os.path.join("$ANDROID_HOME", "tools"),
                                                        os.path.join("$ANDROID_HOME", "platform-tools")]}})
-        UI.delayed_display(DisplayMessage(_("You need to restart your current shell session for your {} installation "
-                                            "to work properly").format(self.name)))
+        UI.delayed_display(DisplayMessage(_(super().POST_INSTALL_WARN).format(self.name)))
```

Comment:
Ah yeah - I wasn't sure, and had tested with `super()` first so went with that. Will update.